### PR TITLE
Learning Lab

### PR DESCRIPTION
**Pull request by debbly**
Addition of the Learning Lab
Screenshot 2023-05-22 at 4 31 16 PM

# Updates
* Lit Protocol header now updated to just `Lit Protocol`
* Top bar nav - has docs, learning lab, and ecosystem
* First lesson - intro to lit

### CODE REVIEW

File: docs/learningLab/intro.md [hunk 1]
```diff
@@ -3,3 +3,13 @@ sidebar_position: 1
 ---
 
 # Introduction to the Learning Lab
+
+![image](../../static/img/learningLab.png)
+
+Welcome to Lit Protocol's Learning Lab! Our curriculum has been designed to empower you with the necessary skills and expertise to excel in building and implementing decentralized access control and programmatic signing. Additionally, by completing each module, you'll have the opportunity to earn exclusive NFTs, demonstrating your expertise in the subject matter.
```
Comment (a1ttech) [inline docs/learningLab/intro.md:9]: "Our curriculum has been designed to empower you with the necessary skills and expertise to excel in building and implementing decentralized access control and programmatic signing."

Perhaps including 'into your own products' (or 'applications') at the end of this sentence. May also want to include hyperlinks to the relevant sections in the main docs tab for people not familiar with these two core services (first exposure to Lit)

File: docs/learningLab/intro.md [hunk 2]
```diff
@@ -3,3 +3,13 @@ sidebar_position: 1
...
+
+Whether you're a curious beginner or an experienced professional aiming to stay at the forefront of this field, the lessons cater to individuals of all proficiency levels. 
+
+You can commence your learning journey by starting with the fundamentals and progressively advancing, or you can directly explore the more complex concepts, pushing the boundaries of your understanding to reach new heights.
+
+Embark on your threshold cryptography adventure today!
```
Comment (a1ttech) [inline docs/learningLab/intro.md:15]: should this last sentence relate to Lit more directly? (our specific implementation of threshold cryptography, not necessarily threshold cryptography in itself)

File: docs/learningLab/introToLit/acc.md [hunk 3]
```diff
@@ -1,10 +1,22 @@
 ---
 sidebar_position: 3
 ---
-# 3. Working with Access Control Conditions 
-ACCs can be harnessed to restrict access to a particular resource (such as a file, webpage, or server). You can set an on-chain condition(s) and then the network will provision signatures and decryption keys for users that meet those conditions.
+# 3. Access Control 
+![image](../../../static/img/ll_acc.png)
 
-### **Example Conditions**
+Access control control conditions can be used to restrict access to a particular resource (such as a file, webpage, or server). You can set an on-chain conditions and then the network will provision signatures and decryption keys for users that meet those conditions.
```
Comment (a1ttech) [inline docs/learningLab/introToLit/acc.md:7]: could be helpful to include a more in depth intro description of ACCs and tying them into decentralized access control / encryption more generally. Emphasizing how all these pieces fit together

File: docs/learningLab/introToLit/acc.md [hunk 4]
```diff
@@ -1,10 +1,22 @@
...
+At completion of this reading you should be able to:
+
+- Describe what access control conditions are useful for.
+- List common types of applications that can be developed with access control.
+
+---
+
+## **Example Conditions**
 
 - User is a member of a DAO
```
Comment (a1ttech) [inline docs/learningLab/introToLit/acc.md:21]: could also be helpful to link to specific examples in the docs here!

File: docs/learningLab/introToLit/acc.md [hunk 5]
```diff
@@ -44,4 +56,29 @@ const accessControlConditions = [
 ]
 ```
 
-To learn more and get started as a developer in the Lit Eco, head on over to our [protocol documentation](https://developer.litprotocol.com/coreConcepts/accessControl/conditionTypes/unifiedAccessControlConditions).
+To learn more and get started as a developer in the Lit ecosystem, head on over to the [protocol documentation](https://developer.litprotocol.com/coreConcepts/accessControl/conditionTypes/unifiedAccessControlConditions).
+
+## Use Cases
```
Comment (a1ttech) [inline docs/learningLab/introToLit/acc.md:61]: just thinking about the flow of information here... perhaps it makes sense to include use cases before diving into particular examples? (ending with a specific CTA to devs who want to dive in and closing out the overview of access control).

File: docs/learningLab/introToLit/intro.md [hunk 6]
```diff
@@ -3,7 +3,7 @@ sidebar_position: 1
 ---
 
 # 1. Intro
-At its core, Lit is an attempt to decentralize [public key cryptography](https://www.cloudflare.com/learning/ssl/how-does-public-key-encryption-work/). This technology underpins all of cryptocurrency and most of the security infrastructure on the web today.
+At its core, Lit is an attempt to decentralize [public key cryptography](https://www.cloudflare.com/learning/ssl/how-does-public-key-encryption-work/). This technology underpins all of cryptocurrency and a majority of the security infrastructure on the web today. A decentralized key management network, Lit can be used in place of centralized key custodians and other key management solutions
 
 Public key infrastructure (PKI) powers two main "buckets" of functionality: encryption and signing.
```
Comment (a1ttech) [inline docs/learningLab/introToLit/intro.md:8]: perhaps adjusting the word choice here this to reflect the recent rework of the 'what is Lit page'. ('Lit provides developers with two core services: encryption and signing')

File: docs/learningLab/introToLit/progSigning.md [hunk 7]
```diff
@@ -1,16 +1,198 @@
...
 ---
-# 5. What can you build with Programmatic Signing
+# 4. Programmatic Signing
+![image](../../../static/img/ll_progSigning.png)
 
 As a distributed key management network, Lit provides developers with the ability to add programmable signing to their applications and wallets. These distributed wallets are known as Programmable Key Pairs (PKP) and the application logic that dictates when and why that key-pair will sign is known as a Lit Action.
 
+---
+
+## Objectives
```
Comment (a1ttech) [inline docs/learningLab/introToLit/progSigning.md:11]: again thinking about the flow of information here. Perhaps considering:
1. Introducing PKPs
2. Introducing Lit Actions
3. How Lit Actions and PKPs work together
4. Use Cases

Moving into a 'working with PKPs and Lit Actions' section (or 'using PKPs and Lit Actions') and including the minting examples and the Lit Action examples

File: docs/learningLab/introToLit/progSigning.md [hunk 8]
```diff
@@ -1,16 +1,198 @@
...
+Use PKPs and Actions to automate your interactions across decentralized finance.
+
+- Condition-based transactions (ex. on-chain limit orders).
+- Recurring payments.
+- Automated vault applications for seamlessly trading asset bundles.
+
+Example Projects:
+
+- [Sling Protocol](https://github.com/Sling-Protocol/pkp-dex-sdk): An SDK for automating transactions on popular DEXs. Currently supports Uniswap V3 and 1inch.
+- [Cask](https://www.cask.fi/): Automated, recurring payments.
```
Comment (a1ttech) [inline docs/learningLab/introToLit/progSigning.md:160]: Cask is using Lit for AC (not PKPs) .. I see that this needs to be updated in the use cases section as well.

**Comment by a1ttech** [comment]
1. The 'Contact Lit Protocol' link needs to be updated (under 'Community Resources' section): https://nut.sh/ell/forms/352580/YEk9vu 

2. Can we use some of our own branding / design elements for the Learning Tab images (intro, access control, and programmatic signing). These images feel a little generic and unauthentic to our brand.

**Comment by a1ttech** [comment]
Perhaps we can include more resources on the Learning Tab intro page? This is the home of the learning section and will be a lot of folks' first exposure to this section of the docs. Could be good to include specific CTAs to other areas and sections of the docs and how to get involved in the protocol in general (maybe adding Ambassador program app?).. could organize it like the docs main 'Overview' landing page with specific callouts to these other resources

File: docs/learningLab/introToLit/intro.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+# 1. Intro
+At its core, Lit is an attempt to decentralize [public key cryptography](https://www.cloudflare.com/learning/ssl/how-does-public-key-encryption-work/). This technology underpins all of cryptocurrency and a majority of the security infrastructure on the web today. A decentralized key management network, Lit can be used in place of centralized key custodians and other key management solutions
+
+Public key infrastructure (PKI) powers two main "buckets" of functionality: encryption and signing.
+
+### **Encryption**
+
+Encryption is the process of encoding information so that it remains hidden or inaccessible to unauthorized parties.
+
+Lit's first feature is a **decentralized access control** protocol interoperable with most EVM chains, Solana, and Cosmos. With Lit, you can use on-chain access control conditions to do 4 main things:
```
Comment (a1ttech) [inline docs/learningLab/introToLit/intro.md:14]: can we simplify this description? We got rid of the 'static' vs 'dynamic' distinction in a previous doc revamp, is there a reason you think this distinction is important to include here?

File: docs/learningLab/introToLit/pok.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+---
+
+# Proof of Knowledge
+![image](../../../static/img/ll_pok.png)
+
+Congratulations on completing this Intro to Lit Protocol lesson! Now it's time to put your newly acquired knowledge to the test and showcase your expertise.
+
+[Go to Gateway.xyz to complete this learning assessment!](https://www.mygateway.xyz/credential/1ec37660-5949-44c4-ad52-0a49f4253877)
+
+This quiz serves as a crucial step in solidifying your understanding and demonstrating your competence in this cutting-edge technologies. It encompasses a range of carefully crafted questions that challenge your comprehension of key concepts, principles, and real-world applications of Lit within the web3 ecosystem.
```
Comment (a1ttech) [inline docs/learningLab/introToLit/pok.md:12]: 'demonstrating your competence in this cutting-edge technologies' -- perhaps changing the word choice here and making more specific to Lit?

File: docs/learningLab/introToLit/pok.md [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+
+By successfully completing the quiz, you will have the chance to earn a Proof of Knowledge credential, validating your proficiency.
+
+Take the plunge and test your knowledge today! Good luck and may your expertise shine through as you dive deeper into the world of decentralized technologies!
```
Comment (a1ttech) [inline docs/learningLab/introToLit/pok.md:16]: same comment as above, might suggest changing to 'dive deeper into the Lit Protocol ecosystem' or something more specific to our product

File: docs/learningLab/introToLit/network.md [hunk 12]
```diff
@@ -0,0 +1,21 @@
+---
+sidebar_position: 2
+---
+
+# 2. How does the network work?
```
Comment (a1ttech) [inline docs/learningLab/introToLit/network.md:5]: 'How does Lit work?'

File: docs/learningLab/introToLit/network.md [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+
+![image](../../../static/img/networkOverview.png)
+
+The Lit network operates through a distributed federation of nodes, where each node plays a vital role in key generation, storage, consensus, and the execution of Lit Actions. However, what sets Lit apart is its reliance on threshold cryptography, which ensures that no single node holds complete control or "executive authority" over the network.
```
Comment (a1ttech) [inline docs/learningLab/introToLit/network.md:9]: Could remove the first sentence ('storage' is somewhat misleading) and just lead off with 'lit is powered by threshold cryptography, meaning no single node holds complete control or "executive authority" over the network' before diving in

File: docs/learningLab/intro.md (hunk 1 above)
Comment (debbly) [reply docs/learningLab/intro.md:9]: I can include the links, but the point of the learning modules is that they will also learn it through the lab readings and exercises

File: docs/learningLab/introToLit/acc.md (hunk 4 above)
Comment (debbly) [reply docs/learningLab/introToLit/acc.md:21]: will add a section to the bottom for addition learning resources!

File: docs/learningLab/introToLit/intro.md [hunk 14]
```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 1
+---
+
+# 1. Intro
+At its core, Lit is an attempt to decentralize [public key cryptography](https://www.cloudflare.com/learning/ssl/how-does-public-key-encryption-work/). This technology underpins all of cryptocurrency and a majority of the security infrastructure on the web today. A decentralized key management network, Lit can be used in place of centralized key custodians and other key management solutions
```
Comment (sarahzdev) [inline docs/learningLab/introToLit/intro.md:6]:
```suggestion
At its core, Lit aims to decentralize [public key cryptography](https://www.cloudflare.com/learning/ssl/how-does-public-key-encryption-work/). This technology underpins all of cryptocurrency and a majority of the security infrastructure on the web today. A decentralized key management network, Lit can be used in place of centralized key custodians and other key management solutions
```

File: docs/learningLab/introToLit/intro.md [hunk 15]
```diff
@@ -0,0 +1,31 @@
...
+
+Public key infrastructure (PKI) powers two main "buckets" of functionality: encryption and programmtic signing.
+
+### Encryption
+
+Encryption is the process of encoding information so that it remains hidden or inaccessible to unauthorized parties.
+
+Lit enables the client-side encryption and decryption of data by setting on or off-chain conditions, without relying on a centralized key custodian.
+
+Lit's **decentralized access control** feature is interoperable with most EVM chains, Solana, and Cosmos. With it you can use on-chain access control conditions to do 4 main things:
```
Comment (sarahzdev) [inline docs/learningLab/introToLit/intro.md:16]:
```suggestion
Lit's **decentralized access control** feature is interoperable with most EVM chains, Solana, and Cosmos. With it you can use on-chain access control conditions to do four main things:
```

File: docs/learningLab/introToLit/intro.md [hunk 16]
```diff
@@ -0,0 +1,31 @@
...
+
+- Encrypt and lock content (images, videos, music, etc) behind an on-chain condition (for example, ownership of an NFT).
+- Decrypt content that was locked behind an on-chain condition.
+- Authorize network signatures that provide access to content (for example, a server or network resource) behind an on-chain condition.
+- Request a network signed JWT that provisions access and authorization to content behind on-chain conditions.
+
+### Programmatic Signing
+
+Lit can be used to program complex signing automations or provide seamless wallet onboarding experiences using Programmable Key Pairs (PKPs) and Lit Actions. PKPs are multi-party computation wallets generated and distributed across the Lit network, while Lit Actions are JavaScript functions dictating when, why, and what a PKP wallet will sign.
```
Comment (sarahzdev) [inline docs/learningLab/introToLit/intro.md:25]:
```suggestion
Lit can be used to program complex signing automations and provide seamless wallet onboarding experiences using Programmable Key Pairs (PKPs) and Lit Actions. PKPs are multi-party computation wallets generated and distributed across the Lit network, while Lit Actions are JavaScript functions dictating when, why, and what a PKP wallet will sign.
```

File: docs/learningLab/introToLit/intro.md [hunk 17]
```diff
@@ -0,0 +1,31 @@
...
+
+Much like in the physical world, digital signatures are used as "proof" that a particular interaction took place. For example, when you swap some tokens on a DEX you must first approve the transaction in your wallet via a signature. Unlike their physical counterparts, cryptographic signatures are tamper-proof and immutable, giving you a secure method of authentication.
```
Comment (sarahzdev) [inline docs/learningLab/introToLit/intro.md:27]: `decentralized exchange` or could remove "on a DEX"

File: docs/learningLab/introToLit/network.md [hunk 18]
```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 2
+---
+
+# 2. The Lit Network
+
+![image](../../../static/img/networkOverview.png)
```
Comment (sarahzdev) [inline docs/learningLab/introToLit/network.md:7]: the img might have a typo. wondering if the copy should be:

"...Shares must **be** combined..."

File: docs/learningLab/introToLit/network.md [hunk 19]
```diff
@@ -0,0 +1,33 @@
...
+
+The Lit network operates through a distributed federation of nodes, where each node plays a vital role in key generation, consensus, and the execution of Lit Actions. What sets Lit apart is its foundation on threshold cryptography. With this approach, Lit ensures that no single node possesses complete control or executive authority over the network. This decentralized structure guarantees enhanced security, resilience, and trustworthiness for all participants in the Lit ecosystem.
+
+
+---
+
+## Objectives
```
Comment (sarahzdev) [inline docs/learningLab/introToLit/network.md:14]: i love this snippet. might be worth having one for 1. Intro page?

File: docs/learningLab/introToLit/network.md [hunk 20]
```diff
@@ -0,0 +1,33 @@
...
+At completion of this reading you should be able to:
+
+- Explain how the Lit network uses threshold cryptgraphy.
+
+---
+
+## Threshold Cryptography
+Threshold cryptography, in the context of Lit, is employed to generate shares of a new public/private key pair, a process known as Distributed Key Generation. Unlike traditional cryptography, the private key of this pair never exists in its entirety. Instead, each node within the network holds a unique private key share.
```
Comment (sarahzdev) [inline docs/learningLab/introToLit/network.md:22]:
```suggestion
Threshold cryptography, in the context of Lit, is employed to generate shares of a new public/private key pair, a process known as Distributed Key Generation (DKG). Unlike traditional cryptography, the private key of this pair never exists in its entirety. Instead, each node within the network holds a unique private key share.
```

File: docs/learningLab/introToLit/acc.md [hunk 21]
```diff
@@ -0,0 +1,88 @@
...
+
+## Objectives
+At completion of this reading you should be able to:
+
+- Describe what access control conditions are useful for.
+- List common types of applications that can be developed with access control.
+
+---
+
+## Example Conditions
```
Comment (sarahzdev) [inline docs/learningLab/introToLit/acc.md:19]: the first two pages have a clear, smooth flow where one is introduced to a concept (e.g., encryption), shown how that feature works in Lit, and then given example ways to use that feature.

for access control, we could explain how access control works and use the example conditions and logic as part of the explanation

File: docs/learningLab/introToLit/acc.md [hunk 22]
```diff
@@ -0,0 +1,88 @@
...
+
+Use on-chain credentials to gate access to your favorite web2 applications and data.
+
+- [Zoom](https://litgateway.com/apps/zoom): Token-gated Zoom calls.
+- [Google Drive](https://litgateway.com/apps/google-drive): Add access control requirements to your Google Drive files.
+- [WordPress](https://litgateway.com/apps/wordpress): Gate access to WordPress sites and pages.
+
+---
+
+## Basic Example
```
Comment (sarahzdev) [inline docs/learningLab/introToLit/acc.md:63]: i like the high-level overview and conceptual learning approach at the start of "intro to lit". the code example and sdk reference may feel abrupt here. we could include a callout to using the sdks at the very end of the section - perhaps as a "next steps" section in the PoK page

Comment (sarahzdev) [reply docs/learningLab/introToLit/acc.md:63]: or to match the next "pkp and lit actions" page, nest "Basic Examples" and "Learn More" under "Working with Access Control"

**Review by sarahzdev — APPROVED** [review]

**Pull request merged**